script/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
Can you add a small README.md file in this folder to show how to use this script? With the different env variables etc

Reply:
Done